wasmplugin/plugin.go

```diff
@@ -313,6 +297,26 @@ func (ctx *httpContext) OnHttpRequestHeaders(numHeaders int, endOfStream bool) t
 		method = string(propMethodRaw)
 	}
 
+	uri := ""
+	if method != http.MethodConnect { // CONNECT requests does not have a path
```

Comment:
could you link a reference on this?

Reply:
Sure, added in the code comment

https://httpwg.org/specs/rfc9110#CONNECT